samples/Office365Groups/README.md

```diff
@@ -0,0 +1,30 @@
+# Office365 Groups Connector
+
+##  Setup
+
+1. [Enable authentication in Azure Active Directory
+](https://docs.microsoft.com/en-us/connectors/custom-connectors/azure-active-directory-authentication#enable-authentication-in-azure-active-directory) to setup authentication for your custom connector via AAD. Keep note of the `AAD application ID` and `Your-Base64-Key`.
+2. Replace `aad_application_id` in apiProperties.json file with your `AAD application ID`.
+3. Use CLI tool `paconn` to create a new custom connector. Enter `Your-Base64-Key` when prompted for the "OAuth2 client secret".
+4. [Set the reply URL in Azure](https://docs.microsoft.com/en-us/connectors/custom-connectors/azure-active-directory-authentication#set-the-reply-url-in-azure), you can find the redirect URL in the security page of your custom connector.
+
+## Supported Actions
+
+The following actions are supported:
+
+* `List all groups`: List all the groups available in an organization.
```

Comment:
Can you please add a bit info on what each operation does and how it works? e.g. we can add parameters does the operation take(if any), behind the scene what graph API gets called, how the response look like etc. Purpose is to educate someone trying to create connector for the first time

Reply:
Will create another PR for this. Checking in current version.